jerry-ext/context/include/jerry-context.h

```diff
@@ -0,0 +1,43 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef JERRYX_CONTEXT_H
+#define JERRYX_CONTEXT_H
+
+#include "jerryscript.h"
+
+#ifndef JERRYX_CONTEXT_SLOTS
+#define JERRYX_CONTEXT_SLOTS 1
+#endif /* !JERRYX_CONTEXT_SLOTS */
+
+/**
+ * Function to pass to jerry_init_with_user_context() as its second parameter
+ */
+void *jerryx_context_init (void);
```

Comment:
you used the prefix  `jerryx_context_` in this ext. But actually, this ext is about mgmt of multiple items (slots) of the user_context of the context, but not the jerry context.

IMO, the name 'user_context' is also confusing, because it is not a context but a user data pointer in the context.

Similarly `context_slot` is also confusing, it is like we have multiple contexts.

My suggestion (draft)
* change `user_context` to `user_data`
* the prefix `jerryx_user_data_slot_`

Make sense?